projects/plugins/jetpack/extensions/shared/components/media-player-control/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarGroup, ToolbarButton } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { ControlBackFiveIcon, ControlForwardFiveIcon } from '../../icons';
+import { STATE_PAUSED, STORE_ID } from '../../../store/media-source/constants';
+import { convertSecondsToTimeCode } from './utils';
+
+function noop () {}
+
+export default function MediaPlayerControl( {
+	skipForwardTime = 5,
+	jumpBackTime = 5,
+	playIcon = 'controls-play',
+	pauseIcon = 'controls-pause',
+	onTimeChange = noop,
+} ) {
+	const {
+		playerState,
+		playerCurrentTime,
+		defaultMediaSource,
+	} = useSelect( select => {
+		const {
+			getMediaSourceCurrentTime,
+			getMediaPlayerState,
+			getDefaultMediaSource,
+		} = select( STORE_ID );
+
+		return {
+			playerState: getMediaPlayerState(),
+			playerCurrentTime: getMediaSourceCurrentTime(),
+			defaultMediaSource: getDefaultMediaSource(),
+		};
+	}, [] );
+
+	const timeInFormat = convertSecondsToTimeCode( playerCurrentTime );
+	const isDisabled = ! defaultMediaSource;
+
+	const { toggleMediaSource } = useDispatch( STORE_ID );
+	const togglePlayer = () => toggleMediaSource( defaultMediaSource.id );
+
+	return (
+		<ToolbarGroup>
+			{ jumpBackTime !== false && (
+				<ToolbarButton
+					icon={ ControlBackFiveIcon }
```

Comment:
Does this icon need to be customizable, since the number of seconds to jump is customizable?

Reply:
yes, not sure how scalable it could be since it will require an icon variation for each custom value. Let's at least make the component customizable.

Reply:
addressed here https://github.com/Automattic/jetpack/pull/18368/commits/1e8157e52580b47c61d3e45278274be897207510